app/src/main/res/values/strings.xml

```diff
@@ -372,4 +372,6 @@
     <string name="never_show_again">Never show again</string>
     <string name="show_manager_update_dialog_on_launch">Show update message on launch</string>
     <string name="show_manager_update_dialog_on_launch_description">Shows a popup notification whenever there is a new update available on launch.</string>
+    <string name="bundle_information">Bundle information</string>
+    <string name="view_patches">View patches</string>
```

Comment:
These strings are for the bundle information dialog. Rename and move them to the other strings of that dialog.